### PR TITLE
add a Jboss troubleshooting doc for classloading errors

### PR DESCRIPTION
**Pull request by obenkenobi**
This provides documentation for troubleshooting using the Java Agent with JBoss EAP and Wildfly

### CODE REVIEW

**Comment by akristen** [comment]
Thanks for opening this PR to make some changes. When it's ready for a tech writer review and a merge, let the team know in #help-documentation. :)

**Review by kford-newrelic — APPROVED** [review]
Great write-up!

File: src/content/docs/apm/agents/java-agent/troubleshooting/jboss-and-wildfly-incomplete-telemetry.mdx [hunk 1]
```diff
@@ -11,25 +11,26 @@ redirects:
   - /docs/agents/java-agent/troubleshooting/jboss-and-wildfly-incomplete-telemetry
 ---
 
-## Problem
+## Problem [#problem]
 
 You are getting incomplete telemetry from an app using JBoss EAP 7.4 (or up) or Wildfly 23 (or up). 
-This could be in the form of web requests being translated into non-web transactions or your dispatcher name is incorrect.
+This can happen in two ways:
```
Comment (obenkenobi) [inline src/content/docs/apm/agents/java-agent/troubleshooting/jboss-and-wildfly-incomplete-telemetry.mdx:17]: There are more ways the telemetry can be incomplete. I'd put them as possible examples.

**Review by akristen — APPROVED** [review]
Thanks!

**Pull request merged**